scripts/build-sw.mjs

```diff
@@ -0,0 +1,18 @@
+import {generateSW} from 'workbox-build';
+
+generateSW({
```

Comment:
for posterity: https://developer.chrome.com/docs/workbox/modules/workbox-build/#generatesw